src/status_im/ui/screens/discover/components/views.cljs

```diff
@@ -48,63 +48,63 @@
+         account-name       :name}                             current-account
+        me?                                                    (= account-address whisper-id)]
+    [react/view
+     [react/view styles/discover-list-item
+      [view/status-view {:id     message-id
+                         :style  styles/discover-item-status-text
+                         :status status}]
+      [react/view styles/discover-list-item-second-row
+       [react/view styles/discover-list-item-name-container
+        [react/view styles/discover-list-item-avatar-container
+         [chat-icon/chat-icon
+          (display-image me? account-photo-path contact-photo-path photo-path whisper-id)
+          {:size 24}]]
+        [react/text {:style           styles/discover-list-item-name
+                     :font            :medium
+                     :number-of-lines 1}
+         (display-name me? account-name contact-name name whisper-id)]]
 
-         (when-not me?
-           (chat-button whisper-id))]
-        (when show-separator?
-          [react/view styles/separator])]])))
+       (when-not me?
+         (chat-button whisper-id))]
+      (when show-separator?
+        [react/view styles/separator])]]))
 
 ;; NOTE(oskarth): Should possibly be merged with discover-list-item-full, but
 ;; there are too many differences between preview item (main screen) and full
 ;; screen, so safer to modify custom styles here without risking regressions.
-(defview discover-list-item-full [{:keys [message show-separator? current-account]}]
-  (letsubs [{contact-name       :name
-             contact-photo-path :photo-path} [:get-in [:contacts/contacts (:whisper-id message)]]]
-    (let [{:keys [name photo-path whisper-id message-id status]} message
-          {account-photo-path :photo-path
-           account-address    :public-key
-           account-name       :name}                             current-account
-          me?                                                    (= account-address whisper-id)]
-      [react/view
-       [react/view styles/discover-list-item-full ;; XXX: Custom style here
-        [view/status-view {:id     message-id
-                           :style  styles/discover-item-status-text
-                           :status status}]
-        [react/view styles/discover-list-item-second-row
-         [react/view styles/discover-list-item-name-container
-          [react/view styles/discover-list-item-avatar-container
-           [chat-icon/chat-icon
-            (display-image me? account-photo-path contact-photo-path photo-path whisper-id)
-            {:size 24}]]
-          [react/text {:style           styles/discover-list-item-name
-                       :font            :medium
-                       :number-of-lines 1}
-           (display-name me? account-name contact-name name whisper-id)]]
+(defview discover-list-item-full [{:keys [message show-separator? current-account contacts]}]
```

Comment:
If we don't need subs here it can be just `defn`

Reply:
indeed, done